server/lorax_server/models/flash_distilbert.py

```diff
@@ -99,22 +121,26 @@ def supports_embeddings(self) -> bool:
     def supports_text_generation(self) -> bool:
         return False
 
-    def warmup(self, batch: FlashEmbeddingBatch, max_new_tokens: int) -> int | None:
+    @property
+    def supports_classification(self) -> bool:
+        return self.classification_head_enabled
+
+    def warmup(self, batch: FlashEmbeddingClassificationBatch, max_new_tokens: int) -> int | None:
         # Note: This is meant to 1) preallocate the memory by doing a forward pass
         # and then just returning the max seqlen since for embeddings we are never generating
-        _ = self.embed(batch)
+        # _ = self.embed(batch)
```

Comment:
Should we delete this and update the comment?

Reply:
good catch